pkg/oc/cli/admin/node/evacuate.go

```diff
@@ -83,42 +82,46 @@ func (e *EvacuateOptions) RunEvacuate(node *kapi.Node) error {
 		return fmt.Errorf("Node '%s' must be unschedulable to perform evacuation.\nYou can mark the node unschedulable with 'oc adm manage-node %s --schedulable=false'", node.ObjectMeta.Name, node.ObjectMeta.Name)
 	}
 
-	labelSelector, err := labels.Parse(e.Options.PodSelector)
+	labelSelector, err := labels.Parse(o.Options.PodSelector)
 	if err != nil {
 		return err
 	}
 	fieldSelector := fields.Set{GetPodHostFieldLabel(node.TypeMeta.APIVersion): node.ObjectMeta.Name}.AsSelector()
 
 	// Filter all pods that satisfies pod label selector and belongs to the given node
-	pods, err := e.Options.KubeClient.Core().Pods(metav1.NamespaceAll).List(metav1.ListOptions{LabelSelector: labelSelector.String(), FieldSelector: fieldSelector.String()})
+	pods, err := o.Options.KubeClient.CoreV1().Pods(metav1.NamespaceAll).List(metav1.ListOptions{LabelSelector: labelSelector.String(), FieldSelector: fieldSelector.String()})
 	if err != nil {
 		return err
 	}
 	if len(pods.Items) == 0 {
-		fmt.Fprint(e.Options.ErrOut, "\nNo pods found on node: ", node.ObjectMeta.Name, "\n\n")
+		fmt.Fprint(o.Options.ErrOut, "\nNo pods found on node: ", node.ObjectMeta.Name, "\n\n")
 		return nil
 	}
-	rcs, err := e.Options.KubeClient.Core().ReplicationControllers(metav1.NamespaceAll).List(metav1.ListOptions{})
+	rcs, err := o.Options.KubeClient.CoreV1().ReplicationControllers(metav1.NamespaceAll).List(metav1.ListOptions{})
 	if err != nil {
 		return err
 	}
 
-	rss, err := e.Options.KubeClient.Extensions().ReplicaSets(metav1.NamespaceAll).List(metav1.ListOptions{})
+	rss, err := o.Options.KubeClient.Extensions().ReplicaSets(metav1.NamespaceAll).List(metav1.ListOptions{})
```

Comment:
We should switch to `Apps` for all workload controllers, this includes ReplicaSet and DaemonSet.

Reply:
And use versioned `AppsV1`